pkgs/applications/accessibility/dasher/default.nix

```diff
@@ -35,7 +35,7 @@ stdenv.mkDerivation {
     echo "true" > build-aux/mkversion
   '';
 
-  configureFlags = lib.optional (!speechSupport) "--disable-speech";
+  configureFlags = lib.optionals (!speechSupport) [ "--disable-speech" ];
```

Comment:
This for example is a noop change and I am not sure if we want stylistic to go that route, or not.

Reply:
I don't think we should include this change in this pr unless we are discouraging/deprecating `lib.optional`